stable/vmware-exporter/OWNERS

```diff
@@ -0,0 +1,10 @@
+approvers:
+- kremers
+- pryorda
+- mtparet
+- chrisz100
+reviewers:
+- kremers
+- pryorda
+- mtparet
+- chrisz100
```

Comment:
I don't know about the helm project but as reviewer permissions are included in approvers the kubernetes best practice is not include people in both approvers AND reviewers. Just put them into approvers as this supersedes.